addon/-private/column-tree.js

```diff
@@ -624,42 +653,68 @@ export default EmberObject.extend({
     this._isSorting = false;
   },
 
-  ensureWidthConstraint() {
+  ensureWidthConstraint(isInitialRun = false) {
```

Comment:
It is worth explaining what `isInitialRun` does 🤔 ✍️

Reply:
Definitely.

Reply:
Refactored away.